src/protocol/resp/src/request/hincrby.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Pelikan Foundation LLC.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use super::*;
+use std::io::{Error, ErrorKind};
+use std::sync::Arc;
+
+counter!(HINCRBY);
+counter!(HINCRBY_EX);
+counter!(HINCRBY_HIT);
+counter!(HINCRBY_MISS);
```

Comment:
I'm not sure that HIT/MISS make sense for this command. At least in how redis-server implements this:
```
If key does not exist, a new key holding a hash is created. If field does not exist the value is set to 0 before the operation is performed.
```